test/unit/model/v2/test_driver_section.py

```diff
@@ -134,13 +134,7 @@ def _model_driver_provider_name_not_nullable_when_vagrant_section_data():
     ['_model_driver_provider_name_not_nullable_when_vagrant_section_data'],
     indirect=True)
 def test_driver_provider_name_not_nullable_when_vagrant_driver(_config):
-    x = {
-        'driver': [{
-            'provider': [{
-                'name': ['unallowed value None', 'null value not allowed']
-            }]
-        }]
-    }
+    x = {'driver': [{'provider': [{'name': ['null value not allowed']}]}]}
```

Comment:
This change is unrelated.

Reply:
I know. It was another issue that blocked the CI.

Reply:
Please avoid touching more than one subject in a PR or it is likely that it will never be merged. Just make another PR for the other fix.

Due to the lack of time I am not even doing reviews that cover move than one area as I know we will not merge them.

Reply:
It is not related to this PR at all. I investigated this PR https://github.com/ansible/molecule/issues/2022 in response to this comment https://github.com/ansible/molecule/pull/2024#issuecomment-488607398 and created this MR https://github.com/ansible/molecule/pull/2025 for that. But, as I already said here https://github.com/ansible/molecule/pull/2024#issuecomment-488635238 I am well aware this is not related.